lib/releng_common/releng_common/__init__.py

```diff
@@ -70,7 +70,12 @@ def create_app(name, extensions=[], config=None, debug=False, debug_src=None,
         if hasattr(app, 'log'):
             app.log.debug('extension `%s` configured.' % extension_name)
 
-    def get_run_options():
+    if redirect_root_to_api:
+        app.add_url_rule("/",
+                         "root",
+                         lambda: redirect(app.api._Api__api.swagger_url))
```

Comment:
Ouch. Looks like you used a "private" attribute here. Do we really need to hide this rule here and make it implicit?